metrics/statsd_test.go

```diff
@@ -48,25 +51,28 @@ func TestStatsdMonitorRecordRun(t *testing.T) {
 		defer client.AssertExpectations(t)
 
 		monitor := metrics.NewStatsdMonitor(client, statsdPrefix)
-		monitor.RecordRun(recipe, duration, false)
+		monitor.RecordRun(recipe, duration, false, 0)
```

Comment:
try to use other number than default value (in this case `0`) to make sure it works fine.

Reply:
Yes, was thinking of a way currently to get the datacount first. btw on this test, since the success is false the count will be 0 right??

Reply:
the count can be more than 0 even when it fails.
but that should not be the point, this module does not care about that, it just focuses on collecting metrics with whatever given.